OpenRA.Mods.Common/Widgets/ButtonWidget.cs

```diff
@@ -207,11 +207,11 @@ public override bool HandleMouseInput(MouseInput mi)
 
 		public override void MouseEntered()
 		{
-			if (TooltipContainer == null || GetTooltipText() == null)
```

Comment:
What is the motivation for these changes?

Reply:
It was crashing with a `System.NullReferenceException`. So I copied the logic from `LabelWithTooltipWidget`. The diffrence is in the null-check (`GetTooltipText()` vs `GetTooltipText`).